README.md

```diff
@@ -1 +1,153 @@
-# booking_api
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [BOOKING API] <a name="about-project"></a>
+
+**[Booking api]** is a front end app to book an automobile. The user can view the cars and their details and add more automobiles. It uses a back end.
+[Link to the back-end](git@github.com:gealsanchez/booking_api.git)
+
+Front-end deployed website: https://wondrous-beijinho-ac8d57.netlify.app/
+Back-end deployed API: https://bookingapi-mstz.onrender.com/api/v1/automobiles --- /users
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[add_automobile]**
+- **[See_details of automobile]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Front-end deployed website: [Live Demo](https://wondrous-beijinho-ac8d57.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [React](https://reactjs.org/) installed and running. To get more information, read the [installation guide](https://react.dev/reference/react).
+
+### Setup
```

Comment:
- [ ] These instructions are meant for the frontend app. This is the backend repo.  Could you replace these instructions with backend instructions?

![image](https://github.com/gealsanchez/booking_api/assets/57421823/4af2537a-34da-4003-8345-ed3f52c22a84)


e.g. bundle install, rails db:create, rspec...